source/class/LevelGeneratorTest.es6

```diff
@@ -0,0 +1,47 @@
+{
```

Comment:
are these enclosing brackets needed?

Reply:
Yes, they provide a block for the "let" commands to be local to; otherwise they would be globals.